README.md

```diff
@@ -309,6 +309,17 @@ All the configurations are grouped in *sections* and some of them can vary depen
   - `OccupancyAlertsMinInterval`:  Sets the desired interval (in seconds) between occupancy alerts.
   - `MaxThreadRestarts`: Defines the number of restarts allowed per thread.
   - `HeatmapResolution`: Sets the resolution used by the heatmap report.
+  - `LogPerformance`: A boolean parameter to enable/disable the logging of "Performance Metrics" in the default processor log.
```

Comment:
What about calling this `LogPerformanceMetrics`?

Reply:
Agree, I changed